tokens/properties/global.json

```diff
@@ -233,6 +233,23 @@
                     "value": "2.500em"
                 }
             },
+            "size-rem": {
```

Comment:
Should we better do:

A. 

`"dimension": {
        "dynamic": {
            "size-em": {
               [...]
                },
            "size-rem": {
               [...]
                },`

B.

`"dimension": {
        "dynamic": {
            "em": {
               [...]
               },
            "rem": {
               [...]
               },`

C.

`"dimension": {
        "dynamic": {
            "size": {
              "em": {
                 }
               },
            "rem": {
               [...]
                },`

Reply:
I vote for option B. Since they're in the "dimension" category they're all size related in some way so the extra "size" prefix doesn't tell us anything new IMO.